sweep/sweeper.go

```diff
@@ -1321,22 +1272,16 @@ func (s *UtxoSweeper) handleUpdateReq(req *updateReq) (
 	newParams.Fee = req.params.Fee
 	newParams.Force = req.params.Force
 
-	log.Debugf("Updating sweep parameters for %v from %v to %v", req.input,
-		pendingInput.params, newParams)
+	log.Debugf("Updating parameters for %v(state=%v) from (%v) to (%v)",
+		req.input, pendingInput.state, pendingInput.params, newParams)
 
 	pendingInput.params = newParams
 
-	// We'll reset the input's publish height to the current so that a new
-	// transaction can be created that replaces the transaction currently
-	// spending the input. We only do this for inputs that have been
-	// broadcast at least once to ensure we don't spend an input before its
-	// maturity height.
+	// We need to reset the state so this input will be attempted again by
+	// our sweeper.
 	//
-	// NOTE: The UtxoSweeper is not yet offered time-locked inputs, so the
-	// check for broadcast attempts is redundant at the moment.
-	if pendingInput.publishAttempts > 0 {
-		pendingInput.minPublishHeight = s.currentHeight
-	}
+	// TODO(yy): a dedicated state?
```

Comment:
No I think resetting to an initial state is ideal if we can get away with it.

Reply:
> imagine the user tries to decrease the fee

I don't think this is possible?

Reply:
> I don't think this is possible?

Yeah, as since we use `testmempoolaccept` now, we'd get an error that the fee increase wasn't high enough.

Reply:
I think it _is_ more correct to to use a new state though as: the input may have already been published with active RBF state by the time a user attempt to do a manual fee bump.